androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -144,7 +144,7 @@ public void init() {
     @Test
     //@SdkSuppress(minSdkVersion = Build.VERSION_CODES.M)
     public void test1_check_certif_not_secure_no_accept()
-            throws InterruptedException, NoSuchFieldException, IllegalArgumentException, IllegalAccessException {
+            throws IllegalArgumentException {
```

Comment:
Lint most likely removed them because there is no function within `oid test1_check_certif_not_secure_no_accept()` that throws those exceptions. If there where the compiler should not accept this.